client/js/components/CardEditorComponent.js

```diff
@@ -8,11 +8,14 @@ if (window.FC === undefined) { window.FC = {}; }
       evt.preventDefault();
 
       var cb = () => {
-        ReactRouter.browserHistory.goBack();
+        // ReactRouter.browserHistory.goBack();
```

Comment:
change this line to

window.alert('you added a card: \r' + this.frontInput.value + " / "+ this.backInput.value);